official/resnet/resnet_run_loop.py

```diff
@@ -344,10 +345,14 @@ def resnet_main(flags, model_function, input_function):
           'multi_gpu': flags.multi_gpu,
           'version': flags.version,
       })
+  benchmark_logger = (logger.BenchmarkLogger(flags.benchmark_log_dir)
+                      if flags.benchmark_log_dir is not None else None)
 
   for _ in range(flags.train_epochs // flags.epochs_between_evals):
     train_hooks = hooks_helper.get_train_hooks(
-        flags.hooks, batch_size=flags.batch_size)
+        flags.hooks,
+        batch_size=flags.batch_size,
+        benchmark_log_dir=flags.benchmark_log_dir)
```

Comment:
Rather than require benchmark_log_dir, I think it would be convenient if it was:
`benchmark_log_dir if benchmark_log_dir is not None else model_dir`
as this makes manual runs with profiling easier.